rbc/heavydb/text_encoding_none.py

```diff
@@ -141,51 +152,24 @@ def heavydb_text_encoding_none_constructor(context, builder, sig, args):
     return heavydb_buffer_constructor(context, builder, sig, args)._getpointer()
 
 
-def get_copy_bytes_fn(module, arr, src_data, sz):
+def get_copy_bytes_fn(builder):
 
-    fn_name = 'copy_bytes_fn'
-    try:
-        return module.get_global(fn_name)
-    except KeyError:
-        pass
+    module = builder.module
 
-    # create function and prevent llvm from optimizing it
-    fnty = ir.FunctionType(ir.VoidType(), [arr.type, src_data.type, sz.type])
-    func = ir.Function(module, fnty, fn_name)
-    func.attributes.add('noinline')
+    name = 'copy_bytes_fn'
+    fnty = ir.FunctionType(void, [i8p, i8p, i64])
+    fn = cgutils.get_or_insert_function(module, fnty, name)
+    fn.linkage = 'linkonce'
```

Comment:
Why `linkonce` here? Can we have multiple definitions of this function in a module?

Reply:
This function is not actually used anymore and we can remove it. But answering your question. Without the linkage as `linkonce`, we would get multiple definition errors.

Reply:
Weird, why so? Also if it's not used anymore then I guess removing it is ok?

Reply:
Yeah, just removed it.